projects/wave-core-new/src/lib/project/project.service.ts

```diff
@@ -790,6 +841,21 @@ export class ProjectService {
         return this.selectedFeature$.value;
     }
 
+    /**
```

Comment:
since this is a static method i guess we can add it to layer or projection utils?

Reply:
eb20aa7a190db2a4aeb01be908dbf358dc155ebf